build.gradle

```diff
@@ -29,16 +35,17 @@ configurations.all {
 // TODO: Specify group, name, description, version of new code being built
 group = 'org.odpi.egeria'
 version = '1.0-SNAPSHOT'
-ext.name = 'New Egeria code'
-description = 'New code for Egeria'
+ext.name = 'egeria-connector-integration-event-schema'
+description = 'Egeria integration connector for Confluent (c) Schema Registry.'
 
 // TODO: Specify Dependency versions as variables
 ext {
-    egeriaversion = '3.6'
+    egeriaversion = '3.14'
     slf4jVersion = '1.7.36'
```

Comment:
these other versions should be updated (as I commented on a previous PR) ie
jackson: 2.14.2
jupiter 5.0.2
httpclient 4.5.14
mockito 4.11.0 (also to v5)

One option may be that most of these can be removed (except egeria) and then import the bom ie
```
         implementation platform("org.odpi.egeria:egeria:${egeriaversion}")
```

This way you should pick up the version constraints of egeria, which I would recommend as the starting point to avoid conflicts (since we don't use sharding or isolating class loaders)

Reply:
I think you mean jupiter 5.9.2? Why should I use an older version?

Reply:
Yes - apologies, typo!